database/factories/UserFactory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use dnj\Ticket\Models\User;
+
+$factory->define(User::class, function (Faker\Generator $faker) {
```

Comment:
This package does not anything do with user.
If you need this for tests, put it in test directory.

Reply:
I put it in the tests/factories folder